src/components/InputPreview.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent, useState } from "react";
+import { Button } from "./atomic";
+
+type PreviewInput = {
+  setFile(file: File | null): void;
+};
+
+const InputPreview = ({ setFile }: PreviewInput) => {
+  const [src, setSrc] = useState("/static/DefaultLogo.svg");
+
+  return (
+    <div className="flex items-center">
+      <img alt={src} className="object-cover rounded-md w-20 h-20" src={src} />
+      <div className="w-10" />
+
+      <Button variant="inverted" size="small">
+        <label>
+          <input
+            onChange={async (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Is this function async?

Reply:
Also, the event type should be inferrable. Does it work if you take out the ChangeEvent<>?